sdk/lib/core/map.dart

```diff
@@ -51,13 +51,25 @@ abstract class Map<K, V> {
   /// A `LinkedHashMap` requires the keys to implement compatible
   /// `operator==` and `hashCode`.
   /// It iterates in key insertion order.
+  /// ```dart
+  /// final data = {1: 'A', 2: 'B', 3: 'C'};
+  /// final mapFrom = Map.from(data);
+  /// print(mapFrom); // {1: A, 2: B, 3: C}
+  /// print(mapFrom.runtimeType); // LinkedHashMap
+  /// ```
   factory Map.from(Map other) = LinkedHashMap<K, V>.from;
 
   /// Creates a [LinkedHashMap] with the same keys and values as [other].
   ///
   /// A `LinkedHashMap` requires the keys to implement compatible
   /// `operator==` and `hashCode`, and it allows `null` as a key.
   /// It iterates in key insertion order.
+  /// ```dart
+  /// final data = {1: 'A', 2: 'B', 3: 'C'};
+  /// final mapOf = Map.of(data);
+  /// print(mapOf); // {1: A, 2: B, 3: C}
+  /// print(mapOf.runtimeType); // LinkedHashMap
```

Comment:
With all of these examples the `runtimeType` has generic params 

So this prints `LinkedHashMap<int, String>` - I think we should be accurate with that?